TWLight/crons.py

```diff
@@ -89,3 +89,17 @@ def do(self):
             management.call_command("djmail_delete_old_messages", days=100)
         except Exception as e:
             capture_exception(e)
+
+
+class UpdateTwlTalkPage(CronJobBase):
```

Comment:
I suggest that we keep the management command, but skip the cron task since we're keeping this in git. Otherwise we'll end up with inconsistencies when we deploy a new update that has an older version of the file. We can use a github action to run the update command instead. That could be a followup patch.

Reply:
Done!